### ISSUE TITLE
 utilities.py/_isnan(values) raised TypeError

### ISSUE THREAD

**Issue by IntenF** [python-visualization/folium#872]
#### _isnan return TyepError
When I tried follows codes, TypeError occured.

```python
map = folium.Map(location=[26.200627, 127.757604])
folium.PolyLine(locations=data[['latitude','longitude']].values).add_to(map)
# data is pandas DataFrame which has 'latitude' and 'logitude'
```
```error
TypeError: only length-1 arrays can be converted to Python scalars```
```
This error was caused by using math module in utilities.py/_isnan ()

``` python
def _isnan(values):
    """Check if there are NaNs values in the iterable."""
    return any(math.isnan(value) for value in _flatten(values))
```

So I suggest you modify it as follows
``` python
def _isnan(values):
    """Check if there are NaNs values in the iterable."""
    return np.isnan(values).any()
```

**Comment by Conengmo** [comment]
Could this be the same as #782? In that case, the problem was in `_flatten()`, which has been fixed already in #758. Can you check which version you are using? If it is 0.5.0, can you install from the Github source?

**Comment by Conengmo** [comment]
I'm closing this issue since this problem has likely already been fixed.
